src/jit/regset.cpp

```diff
@@ -3320,6 +3320,14 @@ void Compiler::tmpPreAllocateTemps(var_types type, unsigned count)
         tmpCount++;
         tmpSize += size;
 
+#ifdef _TARGET_ARM_
+        if (type == TYP_DOUBLE)
+        {
+            // Adjust tmpSize in case it needs alignment
+            tmpSize += TARGET_POINTER_SIZE;
```

Comment:
@hqueue @pgavlin @CarolEidt I investigated what you guys said. But I think the size was not decided exactly in the register allocation phase yet. That's because there was no decided stack offset. So we might have an only just estimate of tmpSize in the preallocation of temp phase.

In this case(TYP_DOUBLE), since the offset would be aligned with multiples of (2*TARGET_POINTER_SIZE), considering the case of aligning by additional TARGET_POINTER_SIZE, It seems to need reserving the TARGET_POINTER_SIZE of additional temp size for aliging beforehand(The final decision of the stack offset is made in Compiler::lvaAllocateTemps. It is only a temporary space for estimating the size. It may not be necessary.).